subjects/fibonacci2/README.md

```diff
@@ -2,12 +2,19 @@
 
 ### Instructions
 
-Complete the body of the function `fibonacci`.
+Complete the body of the **function** `fibonacci`.
 
-This functions receives a number n and returns the nth number in the fibonacci series.
+This functions receives a number `n` and returns the nth number in the fibonacci series.
 
-The Fibonacci Series starts like this: 0, 1, 1, 2, 3, 5, 8, 13... so fibonacci(2) = 1 and fibonnacci(4) = 3
+The Fibonacci Series starts like this: 0, 1, 1, 2, 3, 5, 8, 13 etc...  
+Each number in the series is calculated by the summing of the two previous ones (With the exception of the two first ones which are defined as 0 and 1).
+- so fibonacci(2) = 1 (1 + 1) 
+- and fibonnacci(4) = 3 (1 + 2)
```

Comment:
I don't agree with giving such much information. First, I think they should be able to realize by the sequence of numbers what they need to do. It's fairly obvious that the next number in the sequence is the sum of the previous two. Second, there is so much information online about the fibonacci series: explanations and solutions in every language. You are just giving them the solution.

Reply:
Okeydokey, I ll remove it.

Reply:
Done, I removed those lines. :)